app/src/main/java/com/seafile/seadroid2/data/SeafRepo.java

```diff
@@ -80,6 +88,13 @@ public int getIcon() {
         return R.drawable.repo;
     }
 
+    public boolean canLocalDecrypt() {
+        return encrypted
+                && encVersion >= 2
```

Comment:
will this code still work if Seafile server adds new encVersion 3,4,5,... in the future?